src/test/java/com/moodybluez/enterprise/EnterpriseApplicationTests.java

```diff
@@ -71,7 +71,7 @@ private void thenReturnOneSadMoodForID3() {
 	}
 
 	@Test
-	void saveMoodEntry() throws Exception {
+	void saveMoodEntry_thenValidateAndReturn() throws Exception {
```

Comment:
Again, one more extremely nit-picky kind of change, but I figured that this test could be renamed to be a little bit more specific. From what we've covered in class with TDD and perusing some of the documentation surrounding TDD in java, I believe that making this name change could be another small change just to make things more specific. Here's an article I read on TDD that I found helpful: [SOURCE](https://technologyconversations.com/2013/12/24/test-driven-development-tdd-best-practices-using-java-examples-2/)